### PR TITLE
Update NLP SI example readme

### PR DESCRIPTION
**Pull request by pdmack**
Similar set of changes found elsewhere to account for data dir relocations.
Relates to #232 and #242

### CODE REVIEW

**Review by cwharris — APPROVED** [review]
LGTM

File: examples/nlp_si_detection/README.md [hunk 1]
```diff
@@ -166,11 +166,11 @@ CPP Enabled: True
 ====Starting Pipeline====
 ====Pipeline Started====
 ====Building Pipeline====
-Added source: <from-file-0; FileSourceStage(filename=examples/data/pcap_dump.jsonlines, iterative=False, file_type=FileTypes.Auto, repeat=1, filter_null=True, cudf_kwargs=None)>
+Added source: <from-file-0; FileSourceStage(filename=../data/pcap_dump.jsonlines, iterative=False, file_type=FileTypes.Auto, repeat=1, filter_null=True, cudf_kwargs=None)>
```
Comment (dagardner-nv) [inline examples/nlp_si_detection/README.md:169]: Same as above.

**Review by mdemoret-nv — APPROVED** [review]

File: examples/nlp_si_detection/README.md [hunk 2]
```diff
@@ -111,11 +111,11 @@ morpheus --log_level=DEBUG \
    `# Specify a NLP pipeline with 256 sequence length (Must match Triton config)` \
    pipeline-nlp --model_seq_length=256 \
    `# 1st Stage: Read from file` \
-   from-file --filename=examples/data/pcap_dump.jsonlines \
+   from-file --filename=../data/pcap_dump.jsonlines \
```
[the remark this answers is not in the document]
Comment (pdmack) [reply examples/nlp_si_detection/README.md:114]: Doh! You're absolutely right.

**Comment by pdmack** [comment]
@mdemoret-nv @dagardner-nv PTAL

**Review by mdemoret-nv — APPROVED** [review]

**Review by dagardner-nv — APPROVED** [review]
LGTM, thanks

**Comment by pdmack** [comment]
@gpucibot merge

**Pull request merged**